guides/source/api_app.md

```diff
@@ -433,15 +437,16 @@ Some common modules you might want to add:
 - `AbstractController::Translation`: Support for the `l` and `t` localization
   and translation methods.
 - Support for basic, digest, or token HTTP authentication:
-  * `ActionController::HttpAuthentication::Basic::ControllerMethods`,
-  * `ActionController::HttpAuthentication::Digest::ControllerMethods`,
+  * `ActionController::HttpAuthentication::Basic::ControllerMethods`
+  * `ActionController::HttpAuthentication::Digest::ControllerMethods`
   * `ActionController::HttpAuthentication::Token::ControllerMethods`
 - `ActionView::Layouts`: Support for layouts when rendering.
 - `ActionController::MimeResponds`: Support for `respond_to`.
 - `ActionController::Cookies`: Support for `cookies`, which includes
   support for signed and encrypted cookies. This requires the cookies middleware.
-- `ActionController::Caching`: Support view caching for the API controller. Please notice that
-  you will need to manually specify cache store inside the controller like:
+- `ActionController::Caching`: Support view caching for the API controller. Please note
+  that you will need to manually specify the cache store inside the controller like this:
+
   ```ruby
```

Comment:
Adding a new line here ensures the code fence is properly recognized, but it breaks the block out (and the sentence below it -- "Rails does *not* pass this configuration automatically") out of the bullet point:

![image](https://user-images.githubusercontent.com/771968/89553990-e0559d80-d7d3-11ea-92b2-4af14055066e.png)


To make everything line up correctly, the code block (including the fence) should be indented by an additional two spaces:

![image](https://user-images.githubusercontent.com/771968/89554146-1135d280-d7d4-11ea-8744-c6c5397fec5d.png)